<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="startBTNLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>타이포_쌍문동 B</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>오늘은 뭘 듣고
뭘 먹을까?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startBTN">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>골라줘!</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DX영화자막</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="foodbaseLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DX영화자막 M</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>오늘은

어떠신가요?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoYBTN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="text">
    <string>지금 당장 들으러 가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="foodLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DX영화자막 M</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
